### PR TITLE
PLF-4236: Crowdin synchronization in PLF 3.5.7

### PR DESCRIPTION
**Pull request by maydt**

### CODE REVIEW

File: portlet/exoadmin/src/main/webapp/WEB-INF/classes/locale/portlet/exoadmin/ApplicationRegistryPortlet_es_ES.properties [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+UIAddApplicationForm.header.label=nombre a Visualizar
+UIAddApplicationForm.header.description=Descripci\u00F3n
+UIAddApplicationForm.action.Add=A\u00F1adir
+UIAddApplicationForm.msg.PortletExist=Esta aplicaci\u00F3n ya existe en la categor\u00EDa, por favor seleccione otra aplicaci\u00F3n!
+UIAddApplicationForm.msg.appNotExists=Por favor, selecciona una aplicaci\u00F3n.
+UIAddApplicationForm.msg.typeNoApps=There is no applications of this type.
+UIAddApplicationForm.action.Cancel=#{word.cancel}
+UIAddApplicationForm.label.displayName=#{label.displayName}
+UIAddApplicationForm.label.type=Tipo de Aplicaci\u00F3n
+application.label.=
```
Comment (trangvh) [inline portlet/exoadmin/src/main/webapp/WEB-INF/classes/locale/portlet/exoadmin/ApplicationRegistryPortlet_es_ES.properties:83]: To check if this key is useful.

File: portlet/exoadmin/src/main/webapp/WEB-INF/classes/locale/portlet/exoadmin/ApplicationRegistryPortlet_es_ES.properties [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+##org.exoplatform.applicationregistry.webui.component.UIAddApplicationForm
+UIAddApplicationForm.header.input=Elegir
+UIAddApplicationForm.label.option.portlet=Aplicaci\u00F3n
+UIAddApplicationForm.label.option.gadget=Gadget
+UIAddApplicationForm.header.label=nombre a Visualizar
+UIAddApplicationForm.header.description=Descripci\u00F3n
+UIAddApplicationForm.action.Add=A\u00F1adir
+UIAddApplicationForm.msg.PortletExist=Esta aplicaci\u00F3n ya existe en la categor\u00EDa, por favor seleccione otra aplicaci\u00F3n!
+UIAddApplicationForm.msg.appNotExists=Por favor, selecciona una aplicaci\u00F3n.
+UIAddApplicationForm.msg.typeNoApps=There is no applications of this type.
```
Comment (trangvh) [inline portlet/exoadmin/src/main/webapp/WEB-INF/classes/locale/portlet/exoadmin/ApplicationRegistryPortlet_es_ES.properties:79]: Not yet translated?

File: portlet/exoadmin/src/main/webapp/WEB-INF/classes/locale/portlet/exoadmin/ApplicationRegistryPortlet_es_ES.properties [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+UIOrganizer.msg.emptyCategory=Esta categor\u00EDa no contiene ninguna aplicaci\u00F3n, haz click en el bot\u00F3n (+) para a\u00F1adir una aplicaci\u00F3n.
+UIOrganizer.msg.noCategory=No existen categor\u00EDas. Puedes hacer "click" en el bot\u00F3n "A\u00F1adir Categor\u00EDa" o en el bot\u00F3n "Auto Importar" para a\u00F1adir una categor\u00EDa.
+UIOrganizer.msg.deleteCategoryInUse=No se puede borrar esta categor\u00EDa. Est\u00E1 en uso.
+UIOrganizer.msg.deleteApplicationInUse=No se puede borrar esta aplicaci\u00F3n. Est\u00E1 en uso.
+UIOrganizer.msg.categoryNoExist=Esta categor\u00EDa no existe en la Base de Datos.
+UIOrganizer.msg.applicationNoExist=Esta aplicaci\u00F3n no existe en la Base de Datos.
+
+##org.exoplatform.applicationregistry.webui.component.UIApplicationRegistryEditMode
+UIApplicationRegistryEditMode.title=Importar aplicaciones
+UIApplicationRegistryEditMode.label.showImport=Change Show Import
```
Comment (trangvh) [inline portlet/exoadmin/src/main/webapp/WEB-INF/classes/locale/portlet/exoadmin/ApplicationRegistryPortlet_es_ES.properties:52]: not yet translated?

File: portlet/exoadmin/src/main/webapp/WEB-INF/classes/locale/portlet/exoadmin/ApplicationRegistryPortlet_es_ES.properties (hunk 2 above)
Comment (maydt) [inline portlet/exoadmin/src/main/webapp/WEB-INF/classes/locale/portlet/exoadmin/ApplicationRegistryPortlet_es_ES.properties:79]: Yes, I checked on crowdin, it's not yet translated.

File: portlet/exoadmin/src/main/webapp/WEB-INF/classes/locale/portlet/exoadmin/ApplicationRegistryPortlet_es_ES.properties (hunk 3 above)
Comment (maydt) [inline portlet/exoadmin/src/main/webapp/WEB-INF/classes/locale/portlet/exoadmin/ApplicationRegistryPortlet_es_ES.properties:52]: yes.

File: portlet/exoadmin/src/main/webapp/WEB-INF/classes/locale/portlet/exoadmin/ApplicationRegistryPortlet_es_ES.properties (hunk 1 above)
Comment (maydt) [inline portlet/exoadmin/src/main/webapp/WEB-INF/classes/locale/portlet/exoadmin/ApplicationRegistryPortlet_es_ES.properties:83]: This key is synch from english file.

**Pull request closed without merging**